### PR TITLE
Business Contact Info unit test + Adding id to the ui elements

### PR DESCRIPTION
**Pull request by leksmall**
*Issue #:* /bcgov/entity#2344

*Description of changes:*
Added unit tests for Business Contact Info
Added id to the ui elements

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/DefineCompany/BusinessContactInfo.vue [hunk 1]
```diff
@@ -1,16 +1,23 @@
 <template>
   <div>
     <div v-if="!isEditing">
-      <div>Email Address: {{ !!contact.email ? contact.email : 'Not entered'}}</div>
-      <div>Phone: {{ !!contact.phone ? contact.phone : 'Not entered' }}
+      <div id="lbl-email">Email Address: {{ !!contact.email ? contact.email : 'Not entered'}}</div>
```
Comment (cameron-freshworks) [inline src/components/DefineCompany/BusinessContactInfo.vue]: Just curious, was there a standard set for what we should be using as id names or did you come up with this? Really just asking for my own knowledge :)

Comment (leksmall) [reply src/components/DefineCompany/BusinessContactInfo.vue]: No No :) No standards at all.. i just added it like that

**Comment by leksmall** [comment]
> Code looks good! Though it seems there was an issue with one of the unit tests.

The test runs fine in local. Looks like a timing issue/ value not getting set while running as part of this build. Looking into that

**Comment by severinbeauvais** [comment]
I have confirmed that the unit tests are running successfully in the GHA build.

**Pull request merged**